ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/DropSourceCommand.java

```diff
@@ -16,24 +16,24 @@
 
 package io.confluent.ksql.ddl.commands;
 
-import java.util.Collections;
```

Comment:
Looks like you've got different rules on import layout. Can you switch to Google's style for your KSQL submissions please?

(Raised #1342)